editor/src/components/canvas/controls/classname-select.tsx

```diff
@@ -336,6 +337,27 @@ export const ClassNameSelect = betterReactMemo(
       dispatch([EditorActions.clearTransientProps()], 'canvas')
     }, [updateFocusedOption, dispatch])
 
+    const isMenuOpen = React.useRef(false)
```

Comment:
I like to call out refs in their name, like `isMenuOpenRef`

Reply:
Ah yes, good shout